src/components/add-transaction-dialog/add-transaction-dialog.scss

```diff
@@ -0,0 +1,36 @@
+@import "@material/typography/mdc-typography";
+@import "@material/layout-grid/mdc-layout-grid";
+@import "@material/dialog/mdc-dialog";
+@import "@material/button/mdc-button";
+@import "@material/form-field/mdc-form-field";
+@import "@material/textfield/mdc-text-field";
+@import "@material/radio/mdc-radio";
+@import "@material/menu/mdc-menu";
+@import "@material/select/mdc-select";
+@import "@material/list/mdc-list";
+
+.add-transaction-dialog {
+  &__select-menu {
+    top: inherit !important;
```

Comment:
you should not use important.

Reply:
it is known bug of selects inside dialogs. without "!important" it can be displayed wrong

Reply:
ok